src/main/groovy/io/jenkins/infra/repository_permissions_updater/ArtifactoryPermissionsUpdater.groovy

```diff
@@ -141,10 +141,6 @@ public class ArtifactoryPermissionsUpdater {
                     pathsByGithub[definition.github] = paths
                 }
                 paths.addAll(definition.paths)
-            } else {
-                /* TODO once all have been filled in
-                throw new IOException("$definition.name does not specify the `github` field")
-                */
```

Comment:
Wondering: should we better introduce an exception list to handle this, and this way be able to check this for new and modified in the future?

Reply:
We could, though it is probably overkill. The instructions include the field, so I would hope newly filed entries include it. For the moment the only impact of forgetting it is that you cannot deploy to Essentials until you add it.